lens-doc/lens/private/syntax/srcloc.scrbl

```diff
@@ -0,0 +1,62 @@
+#lang scribble/manual
+
+@(require lens/private/doc-util/main)
+
+@title{Syntax object source locations}
+
+@defthing[syntax-srcloc-lens lens?]{
```

Comment:
Should I document it as `(lens/c syntax? srcloc?)` or `(lens/c syntax? source-location?)`?